service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseRoleServiceTest.java

```diff
@@ -227,7 +230,9 @@ void shouldGrantAccessToAllUsersWithOrganisationId() {
 
         @Test
         void shouldGrantAccessToAllUsersWithoutOrganisationId() {
-            final Set<String> localAuthorityUsers = Set.of(USER_1_ID, USER_2_ID);
+            final Set<String> localAuthorityUsers = new TreeSet<>();
```

Comment:
Can be one lined if you want
```suggestion
            final Set<String> localAuthorityUsers = new TreeSet<>(List.of(USER_1_ID, USER_2_ID));
```